config/logger.js

```diff
@@ -1,17 +1,34 @@
 import { serverEnv } from "@config/schemas/serverSchema";
-import * as PinoLogger from "pino";
+import winston from "winston";
```

Comment:
I thought we were sticking with Pino? What was the benefit to move to Winston?

Reply:
While Pino appends the log messages on a single file continuously, Winston provides the facility of log file rotation or splitting the log file after a specific memory limit has been reached (in this case 10MB). 
It can be beneficial for managing log files, maintaining a manageable file size, and facilitating easier log analysis.

Reply:
As we use Vercel to manage our logs, and in the future possibly Kubernetes not sure those features/reasons would add value

Reply:
I have no idea about Kubernetes. @eddiejaoude would you please suggest on how to move along with the issue?

Reply:
I am trying to understand why we are changing loggers, the reasons you gave do not provide benefit to our project - plus the issue did not say to change loggers

Reply:
@raviraj-github suggested moving to Winston https://github.com/EddieHubCommunity/LinkFree/issues/7437#issuecomment-1587327157. 
 So, I jumped directly into implementing Winston without thinking much about its impact on the project and how it will impact future changes. My bad! 😄

Reply:
I am uncertain about Vercel's log management approach. However, even if we make our application cloud-native ready using Kubernetes, we would still be responsible for managing the application logs. Kubernetes logs differ slightly from application logs.

Nevertheless, to address the current requirements of the issue, we can proceed with Pino and strive to resolve all the mentioned requirements. If necessary, we can later open a new issue with additional analysis in the future.

@akash19coder : Please commit your code with Pino and Single Log file. Thanks for your contribution.